### PR TITLE
Added instructions for vertical aspect direction.

### PR DESCRIPTION
**Pull request by Muzzarino**
I just thought this would be useful to help others use the formula to changing vertical aspect ratio and not only the horizontal one.

### CODE REVIEW

File: docs/api/en/cameras/PerspectiveCamera.html [hunk 1]
```diff
@@ -75,7 +75,26 @@ <h3>[property:Float focus]</h3>
...
+				var h = 9;
+				var aspectlimit = w / h;
+				var aspectYscale = 1;
+				if (.aspect < aspectlimit)
+				{
+					aspectYscale = .aspect / aspectlimit;
+				}
+				
+				// Recalculate the new degrees with the aspect Y scale multiplier.
+				fov = 2 * Math.atan(1 / (1 / Math.tan(fov * Math.PI / 180 / 2) * aspectYscale) ) * 180 / Math.PI;
```
Comment (Mugen87) [inline docs/api/en/cameras/PerspectiveCamera.html]: Can you please link to a source where this formula comes from?

File: docs/api/en/cameras/PerspectiveCamera.html [hunk 2]
```diff
@@ -75,7 +75,26 @@ <h3>[property:Float focus]</h3>
...
+			
+			The [page:.aspect aspect] direction can be vertical using the formula like this:
+			
+			<code>
+				// Calculate the vertical scale if the horizonal aspect is below the specified aspect ratio.
+				var w = 16;
+				var h = 9;
+				var aspectlimit = w / h;
+				var aspectYscale = 1;
+				if (.aspect < aspectlimit)
```
Comment (Mugen87) [inline docs/api/en/cameras/PerspectiveCamera.html]: The variable should be `aspect` (without the dot). Besides, please respect the code style of the project. Especially the spaces insides parens. For example it should look like this:

```js
if ( aspect < aspectlimit ) {

    aspectYscale = aspect / aspectlimit;

}
```

**Comment by WestLangley** [comment]
>I just thought this would be useful to help others use the formula to changing vertical aspect ratio and not only the horizontal one.

Sorry, the docs are not the right place for this.

You can ask a question on the three.js forum and answer your own question if you want to help others.

**Comment by Mugen87** [comment]
Um, I think I agree with @WestLangley.

@Muzzarino There [forum](https://discourse.threejs.org/) offers a "Resources" category that would be perfect for your topic 👍

**Pull request closed without merging**